smart-contracts/contracts/RelayPool.sol

```diff
@@ -475,7 +471,6 @@ contract RelayPool is ERC4626, Ownable {
     // We need to account for it in a streaming fashion
     addToStreamingAssets(feeAmount);
 
-    // TODO: include more details about the origin of the funds
```

Comment:
Hard to do and may not be practical since we may eventually "group" multiple claims.